nx-X11/programs/Xserver/hw/nxagent/Display.c

```diff
@@ -62,7 +62,11 @@ is" without express or implied warranty.
 #include "Init.h"
 #include "Args.h"
 #include "Image.h"
+
+#define Pixmap XlibPixmap
 #include "Icons.h"
+#undef Pixmap
```

Comment:
I didn't quite understand why `#define Pixmap ... #undef Pixmap` is necessary. I see the compile warnings on x86_64, which stem from "Pixmap" being replaced by "Pixmap64", but I haven't found out why exactly. In Agent.h, Pixmap is defined as Pixmap64, but later on undefined again, so it shouldn't come from that.

Not saying the fix is wrong, I just don't understand why "Pixmap" was replaced by "Pixmap64" in the first place.